hack/generate-manifest-flow-visibility.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+_usage="Usage: $0 [--help|-h]
+Generate a YAML manifest for the Clickhouse-Grafana Flow-visibility Solution, using Kustomize, and
+print it to stdout.
+
+This tool uses kustomize (https://github.com/kubernetes-sigs/kustomize) to generate manifests for
+Clickhouse-Grafana Flow-visibility Solution. You can set the KUSTOMIZE environment variable to the
+path of the kustomize binary you want us to use. Otherwise we will look for kustomize in your PATH
+and your GOPATH. If we cannot find kustomize there, we will try to install it."
+
+function print_usage {
+    echoerr "$_usage"
+}
+
+function print_help {
+    echoerr "Try '$0 --help' for more information."
+}
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)    # unknown option
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+source $THIS_DIR/verify-kustomize.sh
+
+if [ -z "$KUSTOMIZE" ]; then
+    KUSTOMIZE="$(verify_kustomize)"
+elif ! $KUSTOMIZE version > /dev/null 2>&1; then
+    echoerr "$KUSTOMIZE does not appear to be a valid kustomize binary"
+    print_help
+    exit 1
+fi
+
+KUSTOMIZATION_DIR=$THIS_DIR/../build/yamls/flow-visibility
+
```

Comment:
Better to add a temp directory here when running `kustomization`, same as we did in other manifest generating scripts, for example https://github.com/antrea-io/antrea/blob/main/hack/generate-manifest-flow-aggregator.sh#L129

Reply:
overlays will be added once we support the option `--mode dev/release`.